README.md

```diff
@@ -74,6 +74,74 @@ The levels are as follows:
 
 ![Data Alignment Analysis]("./../docs/analysis-bnb.png)
 
+
+### Filtering
+To help end users find what they're looking for, we allow filtering on agenda-items as well as sessions in the following ways:
+- [Filter on municipality](#municipality)
+- [Filter on keyword (*)](#keyword)
+- [Filter on date](#date)
+
+*\*: only agenda-items*
+
+These filters are powered by two important moving parts
+1. [mu-cl-resources](https://github.com/mu-semtech/mu-cl-resources): this is the JSON:API our [back-end uses](https://github.com/lblod/app-burgernabije-besluitendatabank) (because JSON:API leaves some freedom of implementation, mentioning this is important! There are a few features and quirks that come from mu-cl-resources itself)
+2. [Ember-Data](https://guides.emberjs.com/release/models/): this is the JSON:API-to-JavaScript our front-end uses
+
+
+#### Default filters
+The following filters are on by default for both agenda-items and sessions:
+- `session:has:governing-body` == true
+- `session.governing-body:has:administrative-unit` == true
+- `session.governing-body.administrative-unit:has:name` == true
+
+
+#### Municipality
+So, the filter we present offers to filter on municipality. This is kind of a lie. Or well, it works different in the back-end.
+
+We chose not to allow filtering on which entity made the decision. While this would allow specific filtering (e.g. "OCMW X" or "Police Y"), this would also:
+- Create a gigantic select options list
+- Make the general use case of "I want to know what is happening in my city" tedious to achieve
+
+Instead, we check `(agenda-item.) session.governing-body.administrative-unit.location`. Our [municipality-list service](app/services/municipality-list.ts) queries all locations on the gemeente/municipality level, and we keep their label/name as well as their id.
```

Comment:
Because this request is used both for agenda-item & session, it means that the starting point is different. The agenda-item requests start from agenda-item and then go to sessions, while the sessions just start from themselves